client/executor/src/integration_tests/mod.rs

```diff
@@ -918,3 +918,14 @@ fn unreachable_intrinsic(wasm_method: WasmExecutionMethod) {
 		error => panic!("unexpected error: {:?}", error),
 	}
 }
+
+test_wasm_execution!(return_value);
+fn return_value(wasm_method: WasmExecutionMethod) {
+	let mut ext = TestExternalities::default();
+	let mut ext = ext.ext();
+
+	assert_eq!(
+		call_in_wasm("test_return_value", &[], wasm_method, &mut ext).unwrap(),
```

Comment:
where is the declaration of that method?

make sure to leave a comment there that the 'return' statement is crucial for that test

Reply:
found it, it was hidden because of the whitespaces

Reply:
https://github.com/paritytech/substrate/pull/12107/files#diff-4cb81001042364564417da256407594e0a778f68fe68a29531931df0951e00b9R356  already there :)